### DISCUSSION TITLE
Fire Event .context Menu Click won't work

### DISCUSSION THREAD

**Discussion by DhineshBabuH** [callstack/react-native-testing-library#1671] [Q&A]
## Ask your Question
Hi,
I am working on righting  test cases for my project so i need to write a test for ContextMenu click so if i have html element and then i if i passed that element to fireEvent.contextMenu(HTMLELement) it won't work so now what do i do ?

**Comment by mdjastrzebski** [comment]
@DhineshBabuH your question contains too few information for anybody to help you. Please describe your challenge clearly and present relevant (focused) code.

**Comment by DhineshBabuH** [comment]
@mdjastrzebski  sir, that issue was resolved i have this new issue,

import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import { TableColumnResizer } from './TableColumnResizer';

test('should handle mouse events correctly', async () => {
  const handleKeyPress = jest.fn(); // Mock the function

  // Render the component with a mock handleKeyPress
  const { container } = render(
    <TableColumnResizer 
      isTableResizer={false} 
      columnIndex={0} 
      rows={rows} 
      updateInitialRowCallback={updateInitialRowCallback} 
    />
  );

  // Get the resize handle
  const resizeHandle = container.querySelector(".resize-handle");


  await userEvent.pointer([{ keys: '[MouseLeft]', target: resizeHandle }]);


  expect(handleKeyPress).toHaveBeenCalled();
});

the event stimulates perfect  but i got the
error like this
    expect(jest.fn()).toHaveBeenCalled()

    Expected number of calls: >= 1
    Received number of calls:    0
Expected Call: The test checks that handleKeyPress was called at least once during the execution of the simulated mouse event.

Received Call Count: The actual call count of handleKeyPress was 0, meaning the function was never invoked during the test.
    
    if i put the debugger on that events in my code it called perfect after the stimulates but the expect condition won't pass ?

**Reply by mdjastrzebski** [reply]
Your issue seems to be with React Testing Library. This repo is React **Native** Testing Library, which is a separate tool. Please log it under proper repository.
